package.json

```diff
@@ -67,4 +68,4 @@
     "build"
   ],
   "type": "module"
-}
+}
```

Comment:
Empty line missing here 🙈

Reply:
Hey, @fboechats! I think we can add a bump command version too.

eg: `pnpm version patch`